packages/website/src/css/custom.css

```diff
@@ -204,6 +204,10 @@ h6 {
   text-decoration: underline;
 }
 
+.markdown a:has(code) {
+  text-underline-offset: 0.25em;
+}
```

Comment:
This was really bugging me...

<table>
<thead>
<tr>
<th>Before</th>
<th>After</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<img alt="Before screenshot of the links in Rule Options Using This Format, with almost no vertical padding above text underlines" src="https://github.com/user-attachments/assets/6f63248d-f238-4dd3-a294-67b21661ae8e" />
</td>
<td>
<img alt="After screenshot of the links in Rule Options Using This Format, with several pixels of vertical padding above text underlines" src="https://github.com/user-attachments/assets/da79ce2d-d24b-4871-93fe-79857f536c21" />
</td>
</tr>
</tbody>
</table>

Reply:
Hmm. I really don't have a preference.